usaspending_api/reporting/tests/integration/test_agencies_overview.py

```diff
@@ -3,18 +3,13 @@
 from django.conf import settings
 from model_mommy import mommy
 from rest_framework import status
-
-from usaspending_api.common.helpers.fiscal_year_helpers import (
-    current_fiscal_year,
-    calculate_last_completed_fiscal_quarter,
-    get_final_period_of_quarter,
-)
+from datetime import datetime, timezone
 
 url = "/api/v2/reporting/agencies/overview/"
 
-CURRENT_FISCAL_YEAR = current_fiscal_year()
-CURRENT_LAST_QUARTER = calculate_last_completed_fiscal_quarter(CURRENT_FISCAL_YEAR) or 1
-CURRENT_LAST_PERIOD = get_final_period_of_quarter(CURRENT_LAST_QUARTER) or 3
+CURRENT_FISCAL_YEAR = 2021
+CURRENT_LAST_QUARTER = 2
```

Comment:
Any reason these were updated to be constant values? Very different from calculating the final period.

Reply:
Wil & I realized this was effectively testing functions that are already tested in their own files. Now that calculate_last_completed_fiscal_quarter is database-dependent, it would require moving all these constants including the assurance_statements below into each test case so it can be marked @pytest.mark.django_db, without adding any value to these tests.